spec/spec_helper.rb

```diff
@@ -10,6 +10,10 @@
 # in spec/support/ and its subdirectories.
 Dir[Rails.root.join("spec/support/**/*.rb")].each {|f| require f}
 
+# Global variable? Sure, why not? Use $in_travis in your tests to, say, exclude tests from running when on Travis CI.
+# NOTE: Then environment variable being checked here is set in .travis.yml.
+$in_travis = !ENV['TRAVIS'].nil? && ENV['TRAVIS'] == 'true'
+
```

Comment:
This works, and probably gives us finer grained control, but another way is with rspec tags. ie, aapb .travis.yml:
```yaml
script: 'bundle exec rspec --tag ~not_on_travis'
```

advantage is that it makes it easier to emulate travis test run without tweaking environment variables.